### ISSUE TITLE
[Accessibility]: Missing data label on Gauge chart + gradient fill color?

### ISSUE THREAD

**Issue by gdimitrov2** [carbon-design-system/carbon-charts#1286]
### Contact Details

georgi.dimitrov95@gmail.com

### Environment

Browser

### What happened? What did you expect to happen instead?

- How can I add a gradient color for the path of the Gauge chart?
- In case some of the values are null, the Gauge charts sets the value 0 instead. How can I add a string to indicate missing data instead of 0?
- How can I add a custom label on the Delta value as well?

### What WCAG 2.1 checkpoint does the issue violate?

Not sure

### Version

0.41.98-d3v5-latest-0.1

### Data & options used

```js
[{
    test: 4,
    test1: 9.2,
    test3: null,
    test4: "inf",
    aggSwAndFwRiskDelta: -13,
  }]

Options:
{
      title: title,
      resizable: true,
      height: height,
      width: "100%",
      color: {
        scale: {
          value: "#87b9f5",
        },
      },
      gauge: {
        type: GaugeTypes.SEMI,
        status: Statuses.WARNING, //danger-red, success-green, warning-yellow
        showPercentageSymbol: true,
      },
    };
```
